LiveChat/index.js

```diff
@@ -377,9 +377,6 @@ export default class LiveChat extends Component {
 	}
 
 	shouldDisableComposer = () => {
-		if (!this.state.onlineStatus && !this.state.chatActive) {
-			return true
-		}
 		if (this.state.queued) {
```

Comment:
@grossvogel @ryanalexanderbrashear What do you all think about instead of removing this completely, we create a new prop that would allow for this to be turned off and on?

Something like 

```javascript
if (!this.state.onlineStatus && !this.state.chatActive && !this.state.allowOfflineMessages) {
			return true
		}

```

Reply:
Yeah that'd be a bit more friendly, like what you have except I think it'd be `!this.props.allowOfflineMessages` (instead of `...state...`)